### PR TITLE
test from 005

### PR DESCRIPTION
**Pull request by zavp005**
test 005

### CODE REVIEW

**Comment by zavp005** [comment]
zzz

File: test-002 [hunk 1]
```diff
@@ -0,0 +1 @@
+test-test-test
```
Comment (zavp005) [inline test-002:1]: good ㅋ

Comment (zavp005) [reply test-002:1]: test zzz

File: .gitignore [hunk 2]
```diff
@@ -0,0 +1,3 @@
+.idea/
+
+testma.iml
```
Comment (zavp005) [inline .gitignore:3]: hm..

**Comment by zavp002** [comment]
asfsa

File: test-002 (hunk 1 above)
Comment (zavp002) [inline test-002:1]: asgsdgfsdfgd

Comment (zavp002) [inline test-002:1]: zzzswwwww

Comment (zavp002) [reply test-002:1]: sdfsd

**Pull request merged**